app/src/features/projects/components/ProjectPermitForm.tsx

```diff
@@ -87,6 +87,16 @@ const ProjectPermitForm: React.FC = () => {
 
   const { values, handleChange, getFieldMeta, errors } = useFormikContext<IProjectPermitForm>();
 
+  const permitTypes = [
+    'Forestry License to Cut',
+    'Heritage Inspection Permit',
+    'License of Occupation',
+    'Occupant License to Cut under the Forest Act',
+    'Park Use Permit',
+    'Road Use Permit',
+    'Special Use Permit'
+  ];
```

Comment:
Just a suggestion, we can use .sort() at the end so we dont have to always worry about putting new permit in lexicographic order.